src/routes/docs/products/auth/mfa/+page.markdoc

```diff
@@ -0,0 +1,1018 @@
+---
+layout: article
+title: Multi Factor authentication
+description: Add multiple layers of authentication to your applications powered by Appwrite Authentication.
+---
+
+{% info title="Unreleased on Cloud" %}
+This feature is not yet released on Cloud, which is on Appwrite `1.4.x`.
+This is a feature available in Appwrite `1.5.x` and will be available on Appwrite Cloud later.
+{% /info %}
+
+Multi-factor authentication (MFA) greatly increases the security of your apps by adding additional layers of protection.
+When MFA is enabled, a malicious actor needs to compromise multiple authentication factors to gain unauthorized access.
+Appwrite Authentication lets you easily implement MFA in your apps, letting you build more securely and quickly.
+
+{% info title="Looking for MFA on your Console account?" %}
+This page covers MFA for your app's end-users.
+If you are looking for MFA on your Appwrite Console account, please refer to the [Console MFA page](/docs/advanced/security/mfa).
+{% /info %}
+
+Appwrite currently allows two factors of authentication. More factors of authentication will be available soon.
+
+Here are the steps to implement MFA in your application.
+
+{% section #display-recover-code step=1 title="Display recovery codes" %}
+
+Initialize your Appwrite SDK's `Client` and `Account`.
```

Comment:
Might also be worth mentioning that we only need avatars for TOTP setup. Or maybe remove it from here, and mention it in TOTP setup only.

Reply:
Done

Reply:
We mentioned it's optional